### PR TITLE
PSR-327

### PR DESCRIPTION
**Pull request by westwater**
Branched from PSR-209

### CODE REVIEW

File: app/pages/nonsipp/landorpropertydisposal/LandOrPropertyStillHeldPage.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import pages.QuestionPage
+import config.Refined.Max5000
+import utils.RefinedUtils._
+import eu.timepit.refined.refineMV
+
+case class LandOrPropertyStillHeldPage(srn: Srn, index: Max5000) extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ toString \ index.arrayIndex.toString
+
+  override def toString: String = "landOrPropertyStillHeld"
```
Comment (kberec) [inline app/pages/nonsipp/landorpropertydisposal/LandOrPropertyStillHeldPage.scala:31]: This is a tricky one. It's the first one where there is an array in an array. Data path is `assets.landOrProperty.landOrPropertyTransactions[].heldPropertyTransaction.disposedPropertyTransaction[n].portionStillHeld`  So multiple disposals can be created for the same heldPropertyTransaction.

File: conf/landorpropertydisposal.routes [hunk 2]
```diff
@@ -2,3 +2,7 @@ GET        /:srn/land-property-disposal/:index                 controllers.nonsi
 POST       /:srn/land-property-disposal/:index                 controllers.nonsipp.landorpropertydisposal.LandOrPropertyDisposalController.onSubmit(srn: Srn, index: Max5000, mode: Mode = NormalMode)
 GET        /:srn/change-land-property-disposal/:index          controllers.nonsipp.landorpropertydisposal.LandOrPropertyDisposalController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = CheckMode)
 POST       /:srn/change-land-property-disposal/:index          controllers.nonsipp.landorpropertydisposal.LandOrPropertyDisposalController.onSubmit(srn: Srn, index: Max5000, mode: Mode = CheckMode)
+GET        /:srn/any-part-land-property-still-held/:index                 controllers.nonsipp.landorpropertydisposal.LandOrPropertyStillHeldController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = NormalMode)
```
Comment (kberec) [inline conf/landorpropertydisposal.routes:5]: Multiple disposals can be created for the same heldPropertyTransaction. Not sure if there is any need to reflect that in the URL, just highlighting it anyway, just for consideration.

Comment (westwater) [reply conf/landorpropertydisposal.routes]: added a double index in the url, can't think of a cleaner solution as we need to propagate both positions when navigating between pages

File: app/pages/nonsipp/landorpropertydisposal/LandOrPropertyStillHeldPage.scala (hunk 1 above)
Comment (westwater) [reply app/pages/nonsipp/landorpropertydisposal/LandOrPropertyStillHeldPage.scala]: changed the path to a double index to capture this

**Review by kberec — APPROVED** [review]

**Pull request merged**